### PR TITLE
Dp 6793 change date update

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/fields/Resources/Resources.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import ResourcesItem from './ResourcesItem/ResourcesItem';
 
 import './resources.scss';
 
+const getNow = () => new Date(Date.now());
```
Comment (royderks) [inline src/fields/Resources/Resources.jsx]: Is this easier for testing?

File: src/widgets/File/File.jsx [hunk 2]
```diff
@@ -85,11 +85,16 @@ class File extends Component {
...
-        if (this.props.registry.formContext && this.props.registry.formContext.setResourceSpecs) {
-          this.props.registry.formContext.setResourceSpecs({
+        const { formContext } = this.props.registry;
+        if (formContext && formContext.setResourceSpecs) {
+          formContext.setResourceSpecs({
             'dcat:byteSize': this.state.total,
             'dcat:mediaType': files[0].type,
-            'ams:distributionType': 'file'
+            'ams:distributionType': 'file',
+            'dct:modified': new Date().toISOString().split('T')[0],
```
Comment (royderks) [inline src/widgets/File/File.jsx]: Why not have the same as getNow() in src/fields/Resources/Resources.jsx?

File: src/fields/Resources/Resources.jsx (hunk 1 above)
Comment (aditudorache) [reply src/fields/Resources/Resources.jsx]: Yes, for mocking

File: src/widgets/File/File.jsx (hunk 2 above)
Comment (aditudorache) [reply src/widgets/File/File.jsx]: Thanks, I wanted to move that to a service already

**Pull request merged**